package-res/ccc/core/base/chart/chart.js

```diff
@@ -599,7 +649,7 @@ def
      * Render the visualization.
      * If not created, do it now.
      */
-    render: function(bypassAnimation, recreate, reloadData) {
+    render: function(bypassAnimation, recreate, dataOnRecreate) {
```

Comment:
Changing the signature this way will break compatibility.
The new "add" mode could only be supported if a new all "keyArgs" interface were used:
```javascript
render: function(keyArgs) {
    var bypassAnimation, recreate, reloadData, addData;
    if(arguments.length === 1 && keyArgs && typeof keyArgs === 'object') {
        bypassAnimation = keyArgs.bypassAnimation;
        recreate = keyArgs.recreate;
        reloadData = recreate && keyArgs.reloadData;
        addData = !reloadData && keyArgs.addData;
    } else {
        bypassAnimation = arguments[0];
        recreate = arguments[1];
        reloadData = arguments[2];
    }
    ...
}
```